code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -23,6 +23,7 @@
 	..()
 	var/jackets = list()
 	jackets["bomber jacket"] = /obj/item/clothing/suit/storage/toggle/bomber
+	jackets["Stellar Corporate Conglomerate jacket"] = /obj/item/clothing/suit/storage/toggle/brown_jacket/scc
 	jackets["corporate black jacket"] = /obj/item/clothing/suit/storage/toggle/leather_jacket/nanotrasen
```

Comment:
Shouldn't these be renamed into NanoTrasen jackets then, or is that best left alone until we actually change the map employer into SCC?